src/database/models/index.js

```diff
@@ -1,35 +1,40 @@
-'use strict';
+import { readdirSync } from 'fs';
+import { basename as _basename, join } from 'path';
+import Sequelize from 'sequelize';
+import dotenv from 'dotenv';
+import { development, production, testing } from '../config/config';
 
-const fs = require('fs');
-const path = require('path');
-const Sequelize = require('sequelize');
-const basename = path.basename(__filename);
+dotenv.config();
+
+const basename = _basename(__filename);
 const env = process.env.NODE_ENV || 'development';
-const config = require(__dirname + '/../config/config.js')[env];
 
-require('dotenv').config();
-console.log(env, config);
+const environment = {
+  development,
+  production,
+  testing
+};
+
+const config = environment[env];
 
 const db = {};
 
-let sequelize;
-if (config.use_env_variable) {
-  sequelize = new Sequelize(process.env[config.use_env_variable], config);
-} else {
-  sequelize = new Sequelize(config.database, config.username, config.password, config);
-}
-
-fs
-  .readdirSync(__dirname)
-  .filter(file => {
-    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
-  })
-  .forEach(file => {
-    const model = sequelize['import'](path.join(__dirname, file));
+const sequelize = new Sequelize(config.use_env_variable, config);
+
+// if (config.use_env_variable) {
+//   sequelize = new Sequelize(process.env[config.use_env_variable], config);
```

Comment:
### Nitpick

- Remove commented code when it's not gonna be used or put a TODO comment on it.